index.js

```diff
@@ -1794,7 +1794,7 @@ module.query = function(context, entity, criteria, callback) {
   }
   if (criteria && typeof criteria !== 'function') {
     url += module.criteriaToString(criteria) || ''
-    url = url.replace(/'/g, '%27').replace(/=/, '%3D').replace(/</, '%3C').replace(/>/, '%3E')
+    url = url.replace(/'/g, '%27').replace(/=/, '%3D').replace(/</, '%3C').replace(/>/, '%3E').replace(/\&/g, '%26');
```

Comment:
Feel free to give it a shot @advait. I think you'll find encodeURIComponent won't work due to the way the Quickbooks backend parses urls.